pkg/userdata/centos/testdata/docker-1.13-kubelet-v1.8-aws.golden

```diff
@@ -3,13 +3,14 @@ hostname: node1
 
 ssh_pwauth: no
 
-
-
 write_files:
 - path: "/etc/sysctl.d/k8s.conf"
   content: |
     net.bridge.bridge-nf-call-ip6tables = 1
     net.bridge.bridge-nf-call-iptables = 1
+    kernel.panic_on_oops = 1
+    kernel.panic = 10
+    vm.overcommit_memory = 1
```

Comment:
This results in `The Linux kernel will always overcommit memory, and never check if enough memory is available. This increases the risk of out-of-memory situations, but also improves memory-intensive workloads.` according to  http://engineering.pivotal.io/post/virtual_memory_settings_in_linux_-_the_problem_with_overcommit - Do we really want that..?

Reply:
I believe yes. That’s what kubelet wants to see, that’s what many memory intensive applications recorded (like JVM workloads or redis).

Reply:
Well I guess the Kubelet will evict pod once the node has memory pressure so presumably this will not cause issues